### DISCUSSION TITLE
Error getting blips.json - have already done blips generate

### DISCUSSION THREAD

**Discussion by Kaleb99jOfficial** [TGRHavoc/live_map-interface#108] [Q&A]
I have followed [this video](https://www.youtube.com/watch?v=0xRuqv7UYlM) and received this error when going to the live map website. (Error getting blips.json. NetworkError when attempting to fetch resource.) I do not understand this as I have done /blips generate while in-server and it said it was making the blips.json file. I waited until 10 minutes after seeing the file stop getting larger through FTP. I have changed the socket port from the port the game is on to a custom port (3000) and it still says the same thing. I have tried using the Nucleus Project proxied server version and the straight-through IP version on the interface and it still has the issue. I do not understand why it is not working.

**Comment by TGRHavoc** [comment]
Don't use the Nucleus proxy and instead set one up yourself. If you do not know how to do this, even with the documentation available, then the easiest solution is to remove HTTPS from your webserver (for the interface) and use the direct IP.

**Comment by Kaleb99jOfficial** [comment]
Marking as completed

I don't remember if I ever fixed this issue, I am not even part of that community anymore. I apologize to the person searching Google for answers.
